deployment/index.md

```diff
@@ -30,17 +31,19 @@ installed on your system.
 
 
 ### Resources
+Here are some resources from the past iteration of Web Dev Weeks:
+
+- [The lab]({{ site.baseurl }}/deployment/lab/)
 
-#### Git
+__Git__
```

Comment:
Any particular reason why you're removing the heading tag? If you want to change the styles it's probably better to just do that in the CSS.